client/src/socket.io.js

```diff
@@ -10,14 +10,21 @@ export default (store) => {
   socket.on('connect', () => {
     // store.dispatch(setSessionId(socket.id));
   });
+
+  socket.on('UPDATE_GAME_LIST', (data) => {
```

Comment:
we could probably do that. Technically, this isn't a redux action, this is the web socket action we are recieving, it just happens that its the same as the action type, and is basically just glue to proxy a call to dispatch that same action.

Thinking through it more, I wonder if every backend websocket action will behave this way. I'm assuming yes, since the websocket actions are basically always used to push some state to redux since thats the only way our components could get that state and update visually somehow. 

If this is true, i think using the actions object as you suggest is good